content/collections/api/en/chart-annotations.md

```diff
@@ -4,122 +4,604 @@ blueprint: api
     {
-        "date": "2023-09-16", 
-        "details": "Added new user properties.", 
-        "id": 50079, 
-        "label": "Version 2.4 Release"
-        } 
-        "success": true
+      "id": 12345,
+      "name": "Alerts"
+    },
+    {
+      "id": 6789,
+      "name": "Releases"
+    }
+  ]
 }
 ```
 
-## Get all chart annotations
+#### Error responses
+
+- `400` - Invalid category
+- `404` - Category not found
 
-Retrieves all chart annotations in your project.
+### Get an annotation category by ID
 
+{{partial:tabs tabs="cURL, HTTP"}}
+{{partial:tab name="cURL"}}
+```bash
+curl --location --request GET 'https://amplitude.com/api/3/annotation-categories/:category_id' \
+-u '{api-key}:{secret-key}'
+```
+{{/partial:tab}}
+{{partial:tab name="HTTP"}}
 ```bash
-GET /api/2/annotations HTTP/1.1
+GET /api/3/annotation-categories/:category_id HTTP/1.1
 Host: amplitude.com
 Authorization: Basic {api-key}:{secret-key} #credentials must be base64 encoded
 ```
+{{/partial:tab}}
+{{/partial:tabs}}
+
+#### Path parameters
 
-### Response
+| Parameter     | Description                                                              |
+| ------------- | ------------------------------------------------------------------------ |
+| `category_id` | <span class="required">Required</span>. Integer. The ID of the category. |
 
+#### Response
+
+```json
+{
+  "data": {
+    "id": 12345,
+    "name": "Alerts"
+  }
+}
+```
+
+#### Error responses
+
+- `400` - Invalid category ID
+- `404` - Category not found
+
+### Update an annotation category
+
+{{partial:tabs tabs="cURL, HTTP"}}
+{{partial:tab name="cURL"}}
+```bash
+curl --location --request PUT 'https://amplitude.com/api/3/annotation-categories/:category_id' \
+--header 'Authorization: Basic {api-key}:{secret-key}' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "category": "Updated Category Name"
+}'
+```
+{{/partial:tab}}
+{{partial:tab name="HTTP"}}
+```bash
+PUT /api/3/annotation-categories/:category_id HTTP/1.1
+Host: amplitude.com
+Authorization: Basic {api-key}:{secret-key} #credentials must be base64 encoded
+Content-Type: application/json
+
+{
+  "category": "Updated Category Name"
+}
+```
+{{/partial:tab}}
+{{/partial:tabs}}
+
+#### Path parameters
+
+| Parameter     | Description                                                                        |
+| ------------- | ---------------------------------------------------------------------------------- |
+| `category_id` | <span class="required">Required</span>. Integer. The ID of the category to update. |
+
+#### Body parameters
+
+| Parameter  | Description                                                                   |
+| ---------- | ----------------------------------------------------------------------------- |
+| `category` | <span class="required">Required</span>. String. The new name of the category. |
+
+#### Response
+
+```json
+{
+   "data": {
+      "id": 569,
+      "category": "updated category name"
```

Comment:
Since the input is capitalized, the output should match `"category": "Updated Category Name"`